images/repmgr/agent/internal/mechanism/native.go

```
@@ -0,0 +1,347 @@
+	if err != nil {
+		return fmt.Errorf("native: read %s: %w", confPath, err)
+	}
+	line := fmt.Sprintf("include '%s'", managedConfName)
+	if strings.Contains(string(b), line) {
+		return nil
+	}
+	f, err := os.OpenFile(confPath, os.O_APPEND|os.O_WRONLY, 0o600)
+	if err != nil {
+		return fmt.Errorf("native: open %s: %w", confPath, err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)